src/components/LandmarkDrawer.tsx

```diff
@@ -152,7 +158,8 @@ function AssessmentModal ({ onClose, assessment }: {
         <p><b>日期：</b>
           {assessment.createdAt && prettifyDate(assessment.createdAt)}</p>
         <p><b>结果：</b>{assessment.score}</p>
-        <p><b>评估人：</b>假评估人</p>
+        <p><b>评估人：</b>{assessment.assessor ?  
+              formatUserName(assessment.assessor.name) : "我" }</p>
```

Comment:
indentation. also, don't repeat yourself (DRY, https://en.wikipedia.org/wiki/Don%27t_repeat_yourself). extract this block into a function and reuse it.